pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1945,6 +1946,27 @@ func handleDelete(ctx *domainContext, key string, status *types.DomainStatus) {
 		status.UUIDandVersion, status.DisplayName)
 }
 
+// lookForRktRunErrors checks for xen errors or any other error which
+// are encountered while rkt run
+// rkt run returns two standard outputs when rkt runs successfully
+// so we need to remove them while checking for errors
+func lookForRktRunErrors(str string) error {
+	var output []string
+	standardOutput1 := "run: group \"rkt\" not found, will use default gid when rendering images"
+	standardOutput2 := "Parsing config from /var/lib/rkt/pods/run/"
+	strSlice := strings.Split(str, "\n")
+	for _, v := range strSlice {
+		if v != standardOutput1 && !strings.HasPrefix(v, standardOutput2) {
+			output = append(output, v)
+		}
+	}
+	errString := strings.Join(output, "\n")
+	if errString != "" {
```

Comment:
So basically.. if you find any other line ( Except the above 2 ) in the output - youa re declaring it as error?

Reply:
Yes